classes/widgets/class-whats-new.php

```diff
@@ -66,31 +66,58 @@ public function the_content() {
 	 * @return array
 	 */
 	public function get_blog_feed() {
-		$feed = \get_site_transient( 'progress_planner_blog_feed_with_images' );
-		if ( false === $feed ) {
+		$feed_data = \get_site_transient( 'progress_planner_blog_feed_with_images' );
```

Comment:
Could we move the `progress_planner_blog_feed_with_images` in a `const` in the object? 🤔 
This way we can avoid repeating it all over the place, and we can change `_site_transient_timeout_progress_planner_blog_feed_with_images` to `'_site_transient_timeout_' . self::TRANSIENT_NAME` to make things clearer

Reply:
Sure, done.